assets/product-compare.scss.liquid

```diff
@@ -0,0 +1,146 @@
+
+$small: 700px;
+$medium: 840px;
+$large: 841px;
+$xlarge: 1000px;
+$xxlarge: 1200px;
+
+.compare-blender {
+.product-table{
+  margin-left:0px;
+}
+.product-wrapper {
+  max-width:80rem;
+  margin:0 auto;
+}
+.product-comparison--bg-image{
+    background-position: 0 50%;
+    background-size: cover;
+    display: inline-block;
+    width: 100%;
+}
+.product-comparison--header-container {
+    padding: 100px 0 120px;
+}
+.product-comparison--sub-leader {
+    font-size: 28px;
+    font-size: 2rem;
+    color: #fff;
+    text-align: center;
+    text-transform: uppercase;
+}
+.product-comparison--leader {
+    font-size: 56px;
+    font-size: 4rem;
+    color: #fff;
+    text-align: center;
+    text-transform: uppercase;
+}
+  #thePopup{
+    width:60%;
+    left: 20%;
+  }
+
+@media screen  and (max-width: $medium) /* Tablet */ {
+  #thePopup{
+    width:80%;
+    left: 10%;
+  }
+
+}
+
+@media screen and (max-width: $small) /* Mobile */ {
+  #thePopup{
+    width:94%;
+    left: 3%;
+  }
+
+}
+#exitButton{
+    background: transparent;
+    padding: 0;
+    border: 0;
+    position:absolute;
+    cursor:pointer;
+    right:10px;
+    top:10px;
+    width:25px;
+}
+#exitButton:hover{
+  opacity:.5;
+}
+#thePopup{
+    background-color: white;
+    height:80%;
+    z-index: 101;
+    top: 100px;
+    position: fixed;
+    overflow:scroll;
+}
+#popupBackground{
+    background-color: black;
+    opacity: .1;
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    z-index: 100;
+}
+  .cell{
+    padding-left: 5px;
+  }
+  .product-borders{
+    border-top-style:solid;
+    border-top-color:#d6d5d5;
+    border-top-width:1px;
+    border-left-style:solid;
+    border-left-color:#d6d5d5;
+    border-left-width:1px;
+  }
+  .bg-gray{
+    background-color: #f5f5f5;
+  }
+  .product-comparison_cell{
+    height:30px;
+  }
```

Comment:
We're starting to create ad hoc typography changes. just wrap this in a header tag and let the native typography settings from the theme takeover.

Reply:
Even if it changes the page from the specified design a little bit, I'm going to be really hard-nosed about this per component styling should be minimal and only addressing markup elements that are unique to that component all other styling such as header styles, text, colors, etc should be pulling from the overarching theme.